src/test/unit_tests/jest_tests/test_nc_nsfs_bucket_cli.test.js

```diff
@@ -161,6 +165,44 @@ describe('manage nsfs cli bucket flow', () => {
             path_exists = await is_path_exists(DEFAULT_FS_CONFIG, bucket_temp_dir_path);
             expect(path_exists).toBe(false);
         });
+
+        it('cli delete bucket not fail with force flag, when bucket path is not empty', async () => {
```

Comment:
```suggestion
        it('cli delete bucket with force flag, when bucket path is not empty', async () => {
```

Reply:
Dont see much change in meaning

Reply:
It is not in the meaning but aligns with the style of the `it` that we have: starts with "should fail" a test that gets an error, otherwise the test is passing (not failing).
[Here](https://github.com/noobaa/noobaa-core/pull/7913/files#diff-978f6ca26a95a48408c53fd0362cbd1bc795a15ee504d248f198a4576a3e745a) I'm trying to add it where tests do not have it.

Reply:
@shirady I dont see any issue with explicitly mentioning a test is not failing in description of a test.

Reply:
Not an issue,  just aligning with the code style of the file.

Reply:
its ok